measurements/views.py

```diff
@@ -4,19 +4,47 @@
 from __future__ import unicode_literals
 
 import os
+import json
 import pathlib
 
-import connexion
+import werkzeug
+from flask import Response, current_app, render_template
+
+from connexion import ProblemException, FlaskApi, Resolver, problem
 
 from measurements.api import api_private_blueprint
 from measurements.pages import pages_blueprint, api_docs_blueprint
 
 HERE = os.path.abspath(os.path.dirname(__file__))
 
+def render_problem_exception(exception):
+    response = exception.to_problem()
+    return FlaskApi.get_response(response)
+
+def render_generic_exception(exception):
+    if not isinstance(exception, werkzeug.exceptions.HTTPException):
+        exc_name = "{}.{}".format(type(exception).__module__,
+                                  type(exception).__name__)
+        exc_desc = str(exception)
+        current_app.logger.warning('Unhandled error occurred, {}: {}'.format(exc_name, exc_desc))
```

Comment:
Is backtrace logged in this case? Seems, it is not, but `KeyError: 'foobar'` is usually not that useful without context.

Reply:
I need to double check this, but I believe the backtrace is logged as part of the flask internals, so this is going to be a log line after the backtrace.